### PR TITLE
Add writing of positions and topology to storage layer

### PR DESCRIPTION
**Pull request by jchodera**
This adds the writing of positions and topologies to the storage layer.

TODO:
- [ ] Add analysis tools to extract trajectories
- [ ] Add movie-making scripts

### CODE REVIEW

**Comment by jchodera** [comment]
We could also store the [potential around each torsion drive for the geometry engine](https://github.com/choderalab/perses/blob/master/perses/rjmc/geometry.py#L857), and then render these using [PyMOL CGOs](http://pymol.sourceforge.net/newman/user/S0500cgo.html#14) to get a better idea of how to improve acceptance rates at the geometry stage.

**Comment by pgrinaway** [comment]
What's the status of this PR?

**Comment by jchodera** [comment]
Writing is all working. The only things that are missing are analysis tools and moviemaking scripts to facilitate doing stuff with the data, but that can certainly something that can be added in a separate PR.

**Comment by pgrinaway** [comment]
Ok. I'll review this PR now. If you want me to merge it, I certainly can. Or, if you want to push more commits, feel free.

File: perses/rjmc/geometry.py [hunk 1]
```diff
@@ -226,6 +227,11 @@ def _logp_propose(self, top_proposal, old_positions, beta, new_positions=None, d
             # DEBUG: Write growth stages
             from simtk.openmm.app import PDBFile
             prefix = '%s-%d-%s' % (self.pdb_filename_prefix, self.nproposed, direction)
+            self._proposal_pdbfile = open("%s-proposal.pdb" % prefix, 'w') # PDB file for proposal probabilities
```
Comment (pgrinaway) [inline perses/rjmc/geometry.py]: I'm curious if we can just write out the sequence of atoms in order. That way, to visualize the introduction, we'd only need to make them visible in the correct order.

Comment (jchodera) [inline perses/rjmc/geometry.py]: That's a great idea!

Comment (jchodera) [inline perses/rjmc/geometry.py]: I've been meaning to remove those PDB-writing blocks and replace them with NetCDF-writing blocks. Should we do that here or in a separate PR?

Comment (pgrinaway) [inline perses/rjmc/geometry.py]: Maybe we can tackle it in a separate PR, since this one is functional?

**Comment by pgrinaway** [comment]
Going to merge this. Improvements can be made in a separate PR.

File: perses/rjmc/geometry.py (hunk 1 above)
Comment (jchodera) [inline perses/rjmc/geometry.py]: Sure! Can you create an issue to remind us of what we wanted to add separately?
